### PR TITLE
Cluster memberships

### PR DESCRIPTION
**Pull request by mrajashree**
Tasks: Managing cluster membership
Tasks: Adding users to clusters, talk about how groups works

### CODE REVIEW

**Comment by MBishop17** [comment]
Making note to add a note per Craig: "not true, we display both, but searching for users when adding them to clusters, projects, or the whitelist of allowed users/groups is limited to only local if you are logged in as local."

File: content/rancher/v2.x/en/tasks/clusters/adding-managing-cluster-members/_index.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
 title: Adding/Managing Cluster Members
 weight: 3525
-draft: true
 ---
-Coming Soon
+
+<h3> Adding a member to a cluster </h3>
+
+You can manage multiple clusters with Rancher. In that, cluster membership lets you specify which users or groups should have access to a particular cluster, and also what permissions they should have within the cluster. 
+Adding a group of users as a cluster member, provides cluster access to all users within that groups. For example, if you have separate developers and QA teams, you can create separate clusters for them, and add them to those clusters respectively.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/adding-managing-cluster-members/_index.md]: we should note that groups only exist for when external auth is used. there is no ability to create local groups (for now)

File: content/rancher/v2.x/en/tasks/clusters/adding-managing-cluster-members/_index.md [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
+
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/adding-managing-cluster-members/_index.md]: we should make a note that you can add clusters while creating clusters (with link to creating clusters), and this documents how to add members to existing clusters

File: content/rancher/v2.x/en/tasks/clusters/adding-managing-cluster-members/_index.md [hunk 3]
```diff
@@ -1,6 +1,32 @@
...
+	3. There is a dropdown to the right of the text box. If external auth provider is configured, this dropdown will be pre-populated with a list of groups that the current logged in user belongs to.  Use the text box to search for and add any other groups.
+4. Once the user/group is selected from the text box, right below it is the `Cluster Permissions` section. Use this to assign users/groups cluster permissions as required while adding them as a member. If a group is added as a member, all users in that group will share the cluster permissions. There are three builtin options:
+	1. **Owner**
+	Owner permission gives full control over the Cluster and all resources inside it. 
+	2. **Member**
+	Members can manage the resources inside the Cluster but not change the Cluster itself. This is the default option
+	3. **Custom**
+	This option allows customization in terms of individual roles. For example, you can choose whether the member should be able to create projects or only view projects, or can manage cluster memberships
+	
+Any **Roles** added previously also show up in the list of available `Cluster Permissions`
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/adding-managing-cluster-members/_index.md]: add link to creating custom roles

File: content/rancher/v2.x/en/tasks/clusters/adding-managing-cluster-members/_index.md [hunk 4]
```diff
@@ -1,6 +1,32 @@
 ---
 title: Adding/Managing Cluster Members
 weight: 3525
-draft: true
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/adding-managing-cluster-members/_index.md]: let's make sure to keep this removed until the final PR to un-draft these docs

**Comment by deniseschannon** [comment]
we should make notes about how if they don't want to see the built-in roles, they should lock them and link to the locking roles PRs.

**Pull request closed without merging**